### PR TITLE
Other resources

### PR DESCRIPTION
**Pull request by betsybookwyrm**
- Adds other resources section as written on cloudstor
- Adds table of contents functionality to pages, enabled only (so far) on the ecosystem page

Would appreciate checking of the TOC functionality/appearance, and your opinions on whether it's a good idea.

### CODE REVIEW

**Review by Quiet27 — CHANGES_REQUESTED** [review]
Looks good! Couple of minor changes suggested. 😸

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr [hunk 1]
```diff
@@ -15,3 +17,45 @@ The ADO ecosystem aims to equip researchers with a set of modular, open source,
...
+
+## What other resources are there for researchers?
+
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
+
+Data collection resources:
+
+-   [twarc](https://twarc-project.readthedocs.io/en/latest/ "https://twarc-project.readthedocs.io/en/latest/"): Command-line utility and Python library for collecting Twitter data
```
Comment (boyd-nguyen) [inline Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:28]: Suggestion: I think UX-wise, the best practice is to open external links in new tabs. (I haven't figured out how to do that in Markdown, so the only way is to use HTML apparently)

Comment (Quiet27) [reply Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:28]: Ah yes, that's a good point. I'm such a 'right click > open in new tab' person that I didn't even think to look at that 😸

**Review by boyd-nguyen — COMMENTED** [review]
It looks good to me. I've put one thing to consider re: external links

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr (hunk 1 above)
Comment (betsybookwyrm) [reply Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:28]: I'm the opposite - I've always felt it was best practice to not specify a link target and allow the user to control their own tabs? I find it annoying myself when something opens in a tab and I don't realise it. Maybe that's just me though?

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr [hunk 2]
```diff
@@ -15,3 +17,45 @@ The ADO ecosystem aims to equip researchers with a set of modular, open source,
...
+
+## What other resources are there for researchers?
+
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
+The ADO Ecosystem is a part of a broader community and ecosystem of open source or openly licensed resources and methods. We’d like to share a list of projects whose work we make use of in our own resources, and which we also frequently recommend to researchers to use either with our workflows or in their own right.
```
[the remark this answers is not in the document]
Comment (betsybookwyrm) [reply Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:24]: oh whoops! I wonder what happened there? I'll fix that.

File: Australian Digital Observatory/content/the-ado-ecosystem/contents.lr (hunk 1 above)
Comment (Quiet27) [reply Australian Digital Observatory/content/the-ado-ecosystem/contents.lr:29]: I've seen some sites have a little message that appears when you hover over a hyperlink that says 'opens in new tab' to let people know before they've clicked. Is it possible to do that?

**Review by Quiet27 — APPROVED** [review]

**Pull request merged**